src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Web/Program.cs

```diff
@@ -1,39 +1,38 @@
-using System;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
 #if (UseNLog)
 using OrchardCore.Logging;
 #endif
 #if (UseSerilog)
 using Serilog;
 #endif
 
-namespace OrchardCore.Templates.Cms.Web
-{
-    public class Program
-    {
-        public static void Main(string[] args)
-        {
-            CreateHostBuilder(args).Build().Run();
-        }
-
-        public static IHostBuilder CreateHostBuilder(string[] args) =>
-            Host.CreateDefaultBuilder(args)
-                .ConfigureLogging(logging => logging.ClearProviders())
-#if (UseSerilog)
-                .UseSerilog((hostingContext, configBuilder) =>
-                    {
-                        configBuilder.ReadFrom.Configuration(hostingContext.Configuration)
-                        .Enrich.FromLogContext();
-                    })
-#endif
-                .ConfigureWebHostDefaults(webBuilder =>
-                {
+var builder = WebApplication.CreateBuilder(args);
+
 #if (UseNLog)
-                    webBuilder.UseNLogWeb();
+builder.Host.UseNLogWeb();
 #endif
-                    webBuilder.UseStartup<Startup>();
+#if (UseSerilog)
+builder.Host.UseSerilog((hostingContext, configBuilder) =>
+                {
+                    configBuilder.ReadFrom.Configuration(hostingContext.Configuration)
+                    .Enrich.FromLogContext();
                 });
-    }
+#endif
+
+builder.Services.AddOrchardCms().AddSetupFeatures("OrchardCore.AutoSetup");
```

Comment:
I don't think we should be adding AutoSetup in the templates.

Reply:
Ok: `AddSetupFeatures("OrchardCore.AutoSetup")` removed.